src/util/json_expr.cpp

```diff
@@ -39,42 +39,45 @@ static exprt simplify_json_expr(
     {
       const irep_idt &value=to_constant_expr(src).get_value();
 
-      if(value!=ID_NULL &&
-         (value!=std::string(value.size(), '0') ||
-          !config.ansi_c.NULL_is_zero) &&
-         src.operands().size()==1 &&
-         src.op0().id()!=ID_constant)
+      if(
+        value != ID_NULL &&
+        (value != std::string(value.size(), '0') ||
+         !config.ansi_c.NULL_is_zero) &&
+        src.operands().size() == 1 &&
+        to_unary_expr(src).op0().id() != ID_constant)
         // try to simplify the constant pointer
-        return simplify_json_expr(src.op0(), ns);
+        return simplify_json_expr(to_unary_expr(src).op0(), ns);
```

Comment:
Please add braces to improve readability. And use `.op()` instead of `.op0()`

Reply:
ok